shared/base/view.js

```diff
@@ -195,7 +195,7 @@ module.exports = BaseView = Backbone.View.extend({
    * Get the HTML for the view, including the wrapper element.
    */
   getHtml: function() {
-    var html = this.getInnerHtml(),
+    var html = this.options.lazy ? '' : this.getInnerHtml(),
```

Comment:
Well, as I tried to represent with the tests, `getHtml` would not return an empty string.  It would always return at least the wrapper element (as I understood the intent of the method to do).  It just wouldn't have any inner html included if it is a lazy-loaded view.